### PR TITLE
Update new workflow for notebooks

### PR DESCRIPTION
**Pull request by alex4200**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,12 +18,18 @@ All the titles, images, descriptions, files, etc are described in [usecases.json
...
+### To deploy in production (non notebook): 
 * The owner receives the confirmation from the tests and the user that everything works correctly.
 * The owner create a pull request from `dev` environment to `master`.
 * A [jenkins plan](https://bbpcode.epfl.ch/ci/job/nse.usecases-wizard/) is triggered.
 * The new changes appear in [Brain Simulation Platform](https://collab.humanbrainproject.eu/#/collab/1655/nav/28538)
 
+### To deploy in production (notebook): 
+
+* The owner is not able to deploy directly a notebook in production.
+* If a notebook has been changed, updated or somehow modified, the owner of THIS repo must be notified.
```
Comment (antonelepfl) [inline README.md]: This point should be after the last one (After testing....)
And I would suggest something like:
```
* If a notebook has been changed, updated or somehow modified, the owner of THIS repo must be notified.
* There is a file `prod_transform_notebooks.py` that needs to be executed in a Collab to update the content of the file keeping the production ID
* If the owners create a new notebook, this ID should be added in the `prod_transform_notebooks.py`
```

File: README.md [hunk 2]
```diff
@@ -18,12 +18,18 @@ All the titles, images, descriptions, files, etc are described in [usecases.json
...
 * The owner receives the confirmation from the tests and the user that everything works correctly.
 * The owner create a pull request from `dev` environment to `master`.
 * A [jenkins plan](https://bbpcode.epfl.ch/ci/job/nse.usecases-wizard/) is triggered.
 * The new changes appear in [Brain Simulation Platform](https://collab.humanbrainproject.eu/#/collab/1655/nav/28538)
 
+### To deploy in production (notebook): 
+
+* The owner is not able to deploy directly a notebook in production.
+* If a notebook has been changed, updated or somehow modified, the owner of THIS repo must be notified. 
+* After testing the existing DEV notebook it will be copied to the PROD environment (the content will be copied to the Collab "Production Notebooks". The 'use_cases.json' in master links to the notebooks on this Collab.)
```
[the remark this answers is not in the document]
Comment (antonelepfl) [reply README.md]: Also add the URL to the Production Collab (https://collab.humanbrainproject.eu/#/collab/9494/nav/71623)

**Comment by antonelepfl** [comment]
Let me know if the changes are ok.

**Comment by alex4200** [comment]
Seems to be ok from my side

**Pull request merged**